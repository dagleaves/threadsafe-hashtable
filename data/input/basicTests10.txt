N 4
I 2 "key2"
I 7 "key7"
I 1 "key1"
I 6 "key6"
I 8 "key8"
D 3
D 6
D 7
L 2
I 0 "key0"
I 4 "key4"
I 9 "key9"
L 9
I 5 "key5"
D 5
D 3
D 5
D 6
I 3 "key3"
L 1
L 2
L 7
L 0
L 9
L 3
L 6
L 0
L 6
L 2
L 6
L 1
L 8
L 7
L 9
L 2
L 0
L 2
