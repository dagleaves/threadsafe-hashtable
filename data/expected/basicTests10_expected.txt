Using 4 threads
OK
OK
OK
OK
OK
Fail
OK
OK
key2
OK
OK
OK
key9
OK
OK
Fail
Fail
Fail
OK
key1
key2
No 7
key0
key9
key3
No 6
key0
No 6
key2
No 6
key1
key8
No 7
key9
key2
key0
key2
